GitUI/HelperDialogs/FormResetCurrentBranch.cs

```diff
@@ -98,5 +115,10 @@ private void Cancel_Click(object sender, EventArgs e)
             DialogResult = DialogResult.Cancel;
             Close();
         }
+
+        private void linkGitReset_LinkClicked(object sender, LinkLabelLinkClickedEventArgs e)
+        {
+            Process.Start(@"https://git-scm.com/docs/git-reset");
```

Comment:
Perhaps you could link directly to the interesting section? https://git-scm.com/docs/git-reset#Documentation/git-reset.txt-emgitresetemltmodegtltcommitgt